task/09-functions-n-closures-tasks.js

```diff
@@ -104,7 +123,21 @@ function memoize(func) {
  * retryer() => 2
  */
 function retry(func, attempts) {
-    throw new Error('Not implemented');
+    return function() {
+        for(let i = 1; i <= attempts; i++) {
```

Comment:
It is necessary to follow the generally accepted style of indentation from operators.

Reply:
I don't understand your comment